.github/workflows/publish-dist.yml

```diff
@@ -87,3 +72,18 @@ jobs:
           # FIXME: make this nicer in tests-scan
           AMQP_SERVER=$(PYTHONPATH=bots python3 -c 'from task.distributed_queue import *; print(DEFAULT_AMQP_SERVER)')
           GITHUB_BASE=${{ github.repository }} bots/tests-scan -d --pull-data "$(cat event.json)" --amqp "$AMQP_SERVER"
+
+      - name: Delete old tags
+        run: |
+          set -ex
+          cd dist-repo
+          now=$(date +%s)
+          for tag in $(git tag -l); do
+              tag_time=$(git show -s --format=%at $tag)
+              if [ $((now - tag_time)) -ge 604800 ]; then
+                  echo "$tag is older than 7 days, deleting..."
```

Comment:
This means that the repo will be totally empty after a week when nobody pushes new tags in the mean time, no? What about "keep the N most recent commits" where N is such that we are happy with the size of the repo.

Reply:
Good point. I'll change it to at least retain the current cockpit-machines main HEAD, so that this will always stay available.

Reply:
FTR, it will never be *totally* empty -- after all, this publish-dist workflow just pushed a new tag. But that may be for a PR only, and I agree it's kind of nice to keep main HEAD around.